packaging/package/package.py

```diff
@@ -12,44 +13,53 @@
 
 
 def create_arg_parser():
-    parser = argparse.ArgumentParser(description="Tableau Connector Packaging Tool: package and sign connector files into a single Tableau Connector (" + PACKAGED_EXTENSION + ") file.")
+    parser = argparse.ArgumentParser(description="Tableau Connector Packaging Tool: package and sign connector files into a single Tableau Connector (" + PACKAGED_EXTENSION + ") file.")  # noqa: E501
```

Comment:
What is #noqa: e501 mean? Is it telling the PEP bot to ignore this line or something?

Reply:
yup. you can do a plain `# noqa`, including the pep8 error code is optional but makes sure it ignores what you're intending.